webpack/scenes/ContentViews/Details/Versions/Delete/RemoveSteps/CVVersionRemoveReview.js

```diff
@@ -58,29 +64,91 @@ const CVVersionRemoveReview = () => {
       {affectedHosts &&
         <>
           <h3>{__('Content hosts')}</h3>
-          <Flex>
-            <FlexItem><ExclamationTriangleIcon /></FlexItem>
-            <FlexItem><p>{__(`${pluralize(hostResponse.length, 'host')} will be moved to content view ${selectedCVNameForHosts} in `)}</p></FlexItem>
-            <FlexItem><Label isTruncated color="purple" href={`/lifecycle_environments/${selectedEnvForHost[0].id}`}>{selectedEnvForHost[0].name}</Label></FlexItem>
-          </Flex>
+          {singleCVHostsCount > 0 && (
+            <Flex>
+              <FlexItem><ExclamationTriangleIcon /></FlexItem>
+              <FlexItem data-testid="single-cv-hosts-remove">
+                <FormattedMessage
+                  id="single-cv-hosts-remove"
+                  defaultMessage="{count, plural, one {{singular}} other {# {plural}}} will be moved to content view {cvName} in {envName}."
```

Comment:
This is omitting the number:
![image](https://github.com/user-attachments/assets/7e1f2e83-1308-4860-af3a-7887f2fa6520)